### PR TITLE
feat: support config from the dashboard

### PR DESCRIPTION
**Pull request by royhadad**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -81,8 +81,8 @@ func start(port string) {
 	state.SetClusterUuid(clusterUuid)
 	state.SetK8sVersion(k8sVersion)
 
-	server.InitServerVars()
-	certPath, keyPath, err := server.ValidateCertificate()
+	skipList.InitSkipList()
+	certPath, keyPath, err := skipList.ValidateCertificate()
```
Comment (TzlilSwimmer123) [inline main.go:85]: `ValidateCertificate()` has 0 connection to skipList. It lives w/o it

File: pkg/clients/cliServiceClient.go [hunk 2]
```diff
@@ -62,30 +62,45 @@ func NewCustomCliServiceClient(baseUrl string, httpClient HTTPClient, timeoutCli
 	}
 }
 
-func (c *CliClient) RequestEvaluationPrerunData(tokenId string) (*cliClient.EvaluationPrerunDataResponse, error) {
+type ClusterEvaluationPrerunDataResponse struct {
+	cliClient.EvaluationPrerunDataResponse `json:",inline"`
```
Comment (TzlilSwimmer123) [inline pkg/clients/cliServiceClient.go:66]: `json:",inline"` interesting!

File: pkg/clients/cliServiceClient.go [hunk 3]
```diff
@@ -62,30 +62,45 @@ func NewCustomCliServiceClient(baseUrl string, httpClient HTTPClient, timeoutCli
 	}
 }
 
-func (c *CliClient) RequestEvaluationPrerunData(tokenId string) (*cliClient.EvaluationPrerunDataResponse, error) {
+type ClusterEvaluationPrerunDataResponse struct {
```
Comment (TzlilSwimmer123) [inline pkg/clients/cliServiceClient.go:65]: what is the idea behind creating a new struct?

**Review by shalev007 — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (royhadad) [reply main.go]: agreed, and reverted to the previous code. in theory, they should get split into 2 separate packages

**Pull request merged**